templates/ansible/resource.erb

```diff
@@ -266,15 +266,30 @@ def update_fields(module, request, response):
 
 <% update_props.each do |key, props| -%>
 <% func_name = "#{props.first.name.underscore}_update" -%>
+<% fingerprint_name = props.map { |x| x.fingerprint_name }.compact.first -%>
```

Comment:
Instead of mapping to props, I think we can add `fingerprint_name` to the `:old` case in `properties_by_custom_update`, at which point it will be available in `key`